geode-wan/src/test/java/org/apache/geode/cache/wan/internal/WanCopyRegionFunctionServiceTest.java

```diff
@@ -145,7 +145,7 @@ public void cancelAllExecutionsWithRunningExecutionsReturnsCanceledExecutions()
         });
 
     // Wait for the functions to start execution
-    await().untilAsserted(() -> assertThat(service.getNumberOfCurrentExecutions()).isEqualTo(2));
+    await().untilAsserted(() -> assertThat(service.getNumberOfCurrentExecutions()).isLessThanOrEqualTo(2));
```

Comment:
I think with this change the test would continue too quickly.

The goal of this line is to wait until the tasks have started. `await().untilAsserted` repeatedly runs `() -> assertThat(service.getNumberOfCurrentExecutions()).isEqualTo(2)` until it succeeds (or times out).

Changing the assertion to `.isLessThanOrEqualTo(2)` would make the test continue before both tasks have started, which I think is not what we want.

Do you see it time out? Perhaps we need to give it some more time before it gives up waiting for both tasks to start?